config/mail.php

```diff
@@ -84,8 +84,8 @@
     */
 
     'from' => [
-        'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
-        'name' => env('MAIL_FROM_NAME', 'Example'),
+        'address' => env('MAIL_FROM_ADDRESS', 'hoydonamos@gmail.com'),
+        'name' => env('MAIL_FROM_NAME', 'Che, ¿Hoy donamos?'),
```

Comment:
Does it are added on an env file? And this date (the second param of the `env()`) is loaded on fallback if the env is not found?

Reply:
These already exits in the env file, and yes, if it doesnt work in the env dile would use the second param. I added it beacuse the doc of mail ask to do it.